### DISCUSSION TITLE
A new build should stop a running build

### DISCUSSION THREAD

**Discussion by ghost** [coollabsio/coolify#1205] [Feature Requests]
Use case: I am changing a lot of stuff in a CMS in a short period of time. This triggers a new build each time. So when the first build is finished, the job should be done, but then the builds in the queue will start, even if it’s already done. Or in other words: A new build does not stop a running build.

Alternatively we could replace the next queued build as long as the running build has not finished. This could be an easier solution …

**Comment by andrasbacsai** [comment]
Is this related to v3?
